nums/numpy/api.py

```diff
@@ -246,6 +261,10 @@ def atleast_3d(*arys):
 
 
 def arange(start=None, stop=None, step=1, dtype=np.int64) -> BlockArray:
```

Comment:
Can you confirm the semantics for numpy? If we pass floats to this function, does it work or raise an exception?

Reply:
Yes, it seems like it rounds up to the nearest whole number, so `np.arange(3.1)` outputs `[1, 2, 3, 4]`. I will adjust this implementation to account for this.

Reply:
Sure.

Reply:
Update: Seems like `arange` might have a bug. `nps.arange(3.1, 5.1)` should return `[3.1 4.1]`, but it currently returns `[0, 1]` (without me changing anything). Will look into this.

Reply:
Resolved offline.